modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/GiveRatingScreen.kt

```diff
@@ -0,0 +1,198 @@
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Star
+import androidx.compose.material.icons.filled.StarBorder
+import androidx.compose.material.icons.filled.StarHalf
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.bars.NavigationButton
+import au.com.shiftyjelly.pocketcasts.compose.bars.NavigationIconButton
+import au.com.shiftyjelly.pocketcasts.compose.buttons.RowButton
+import au.com.shiftyjelly.pocketcasts.compose.components.PodcastCover
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.GiveRatingViewModel
+import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.GiveRatingViewModel.State.Loaded.Stars
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun GiveRatingScreen(
+    state: GiveRatingViewModel.State.Loaded,
+    setStars: (Stars) -> Unit,
+    submitRating: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+
+    val context = LocalContext.current
+    Content(
+        state = state,
+        setStars = { setStars(it) },
+        submitRating = {
+            Toast.makeText(context, "TODO: Submit rating", Toast.LENGTH_LONG).show()
+            submitRating()
+        },
+        onDismiss = onDismiss,
+    )
+}
+
+@Composable
+private fun Content(
+    state: GiveRatingViewModel.State.Loaded,
+    setStars: (Stars) -> Unit,
+    submitRating: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxSize()
+    ) {
+        NavigationIconButton(
+            iconColor = MaterialTheme.theme.colors.primaryText01,
+            navigationButton = NavigationButton.Close,
+            onNavigationClick = onDismiss,
+            modifier = Modifier.align(Alignment.Start)
+        )
+
+        Spacer(Modifier.weight(1f))
+
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+
+            PodcastCover(
+                uuid = state.podcastUuid,
+                coverWidth = 164.dp,
+            )
+
+            Spacer(Modifier.height(40.dp))
+
+            TextH30(text = stringResource(LR.string.podcast_rate, state.podcastTitle))
```

Comment:
This text needs center-alignment and horizontal padding.

 
![image](https://github.com/Automattic/pocket-casts-android/assets/1405144/3f932b19-3ca2-4a25-8c5a-c113b4258fee)

Reply:
Yes, yes it certainly does. Added in https://github.com/Automattic/pocket-casts-android/pull/1451/commits/de5bf3cb64df6e3536ff1b422d974e2365c8c73b